app/src/main/kotlin/com/wire/android/ui/home/conversations/model/messagetypes/multipart/standalone/VideoAssetPreview.kt

```diff
@@ -0,0 +1,238 @@
+    val height = item.metadata?.height() ?: 0
+    val maxWidth = calculateMaxMediaAssetWidth(
+        item = item,
+        maxDefaultWidth = dimensions().attachmentVideoMaxWidth,
+        maxDefaultWidthLandscape = dimensions().attachmentVideoMaxWidthLandscape
+    )
+
+    Column(
+        modifier = Modifier
+            .widthIn(max = maxWidth)
+            .background(
+                color = colorsScheme().surface,
+                shape = RoundedCornerShape(dimensions().messageAttachmentCornerSize)
+            )
+            .border(
+                width = dimensions().spacing1x,
+                color = colorsScheme().outline,
+                shape = RoundedCornerShape(dimensions().messageAttachmentCornerSize)
+            )
+            .clip(RoundedCornerShape(dimensions().messageAttachmentCornerSize))
+            .padding(dimensions().spacing10x),
+        horizontalAlignment = Alignment.CenterHorizontally,
+        verticalArrangement = Arrangement.spacedBy(dimensions().spacing8x)
+    ) {
+
+        FileHeaderView(
+            modifier = Modifier.padding(dimensions().spacing8x),
+            extension = item.mimeType.substringAfter("/"),
+            size = item.assetSize,
+        )
+
+        Box(
+            modifier = Modifier
+                .aspectRatio(aspectRatio(width, height))
+                .background(
+                    color = colorsScheme().outline,
+                    shape = RoundedCornerShape(dimensions().messageAttachmentCornerSize)
+                )
+                .border(
+                    width = 1.dp,
+                    color = colorsScheme().outline,
+                    shape = RoundedCornerShape(dimensions().messageAttachmentCornerSize)
+                )
+                .clip(RoundedCornerShape(dimensions().messageAttachmentCornerSize)),
+            contentAlignment = Alignment.Center
+        ) {
+
+            // Video preview image
+            if (item.previewAvailable()) {
+                AsyncImage(
+                    modifier = Modifier.fillMaxSize(),
+                    model = item.previewImageModel(
+                        decoderFactory = { result, options, _ ->
+                            VideoFrameDecoder(result.source, options)
+                        }
+                    ),
+                    contentScale = ContentScale.Crop,
+                    contentDescription = null,
+                )
+            }
+
+            // Video duration text
+            item.metadata?.durationMs()?.let {
+                Text(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .align(Alignment.BottomEnd)
+                        .background(color = colorsScheme().scrim)
+                        .padding(dimensions().spacing4x),
+                    text = DateAndTimeParsers.videoMessageTime(it),
+                    style = typography().subline01,
+                    color = Color.White,
```

Comment:
It needs to always be white because it's drawn on `scrim` background, right?
In that case, instead of hardcoding this color, you can add it to `WireFixedColorScheme` (it's the scheme for colors that are always the same no matter what theme is currently used) or use directly dark palette like we do for calling `ParticipantTile` (it's always drawn on dark background so it needs dark color scheme even for light theme), so in this case probably `darkColorsScheme().onSurface`